### PR TITLE
Travis: Linux wxPython Wheel

### PR DESCRIPTION
**Pull request by ax3l**
Fix wxPython build by providing the GTK3 dependency of wxPython 4.

Also loads the proper wheel as recommended in the docs:
https://wxpython.org/pages/downloads/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,7 +68,11 @@ install:
   # python packages
   - conda install -q h5py lxml numpy matplotlib pydap requests shapely proj4 geos cartopy setuptools gdal
   - pip install --no-deps hyo.bag
-  - pip install wxPython
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        pip install wxPython;
+    else
+        pip install -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-14.04 wxPython;
```
Comment (giumas) [inline .travis.yml:74]: This must be updated when Appveyor moves to a newer Ubuntu's release. Correct?

**Comment by giumas** [comment]
Thanks!

File: .travis.yml (hunk 1 above)
Comment (ax3l) [reply .travis.yml:74]: Yes.
This happens for (Travis-CI) like once in 10 years.

**Comment by ax3l** [comment]
@giumas let us please update the readme & requirements.txt to `wxPython "Phoenix" >= 4.0.0` since this is the only available release line and it works

**Comment by giumas** [comment]
Sure, go ahead!

**Pull request merged**